lib/avo/concerns/fetches_things.rb

```diff
@@ -51,6 +51,9 @@ def get_resource_by_name(name)
         #
         # get_resource_by_name('User') => UserResource
         # get_resource_by_name(User) => UserResource
+
+        # FIXME: This query doesn't necessarily get the right resource.
+        # From all the resources that use the same model_class, it will fetch the first one in alphabetical order
```

Comment:
`"it will fetch the first one in alphabetical order"`

It happens when `model_resource_mapping` is missing.  https://docs.avohq.io/2.0/resources.html#use-multiple-resources-for-the-same-model 

Anyway, i think `model_resource_mapping` don't solve the problem addressed in this PR because it only allows you to choose 1 default resource per model.

Reply:
Exactly ; I saw this configuration, unfortunately it won't be of help in our case. Our use case of Avo is building multiple and specific resources around one main model, which is tied to probably a dozen resources...